### DISCUSSION TITLE
Brew installation

### DISCUSSION THREAD

**Discussion by molere** [Homebrew/discussions#4820] [Casks]
### Output of `brew config`

```shell
==> Next steps:
- Run these two commands in your terminal to add Homebrew to your PATH:
    (echo; echo 'eval "$(/usr/local/bin/brew shellenv)"') >> /Users/star/.zprofile
    eval "$(/usr/local/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

star@stars-iMac ~ %
```

### Output of `brew doctor`

```shell
only this
```

### Description of issue

   How to type thesse commands exactly in my `Terminal`

`==> Next steps:
- Run these two commands in your terminal to add Homebrew to your PATH:
    (echo; echo 'eval "$(/usr/local/bin/brew shellenv)"') >> /Users/star/.zprofile
    eval "$(/usr/local/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

star@stars-iMac ~ %

### Relevant casks

only this

**Comment by SMillerDev** [comment]
Exactly like they appear 
```sh
(echo; echo 'eval "$(/usr/local/bin/brew shellenv)"') >> /Users/star/.zprofile
```
```sh
eval "$(/usr/local/bin/brew shellenv)"
```
